server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java

```diff
@@ -184,6 +184,10 @@ Statistics currentFileStatistics() {
     return firstTimeSeriesMetadata.getStatistics();
   }
 
+  boolean currentFileModified() {
+    return firstTimeSeriesMetadata.hasModification();
```

Comment:
if firstTimeSeriesMetadata is null, throw exception

Reply:
fixed